onmt/modules/Embeddings.py

```diff
@@ -0,0 +1,147 @@
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+from onmt.modules import BottleLinear
+from onmt.modules import aeq
+
+
+class Elementwise(nn.ModuleList):
```

Comment:
Could put this in modules/utils if you think it is useful in other locations.

Reply:
I considered this. I can't think of any other cases where it's useful yet, but it wouldn't be surprising to me if there were some. I'll devote some thought to this and other places in the code where containers could be useful soon.